pkg/cloud/services/eks/errors.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eks
+
+import "github.com/pkg/errors"
+
+var (
+	// ErrClusterExists is an error if a EKS cluster already exists with
+	// the same name in the spec but that isn't owned by the CAPI cluster
+	ErrClusterExists = errors.New("eks cluster already exists with same name but isn't owned by cluster")
+	// ErrUnknowTokenMethod defines an error if a unsupported token generation method is supplied
+	ErrUnknowTokenMethod = errors.New("unknown token method")
```

Comment:
```suggestion
	// ErrUnknownTokenMethod defines an error if a unsupported token generation method is supplied
	ErrUnknownTokenMethod = errors.New("unknown token method")
```

And fix up the consumers.

Reply:
Thanks @michaelbeaumont for fixing. I also just fixed a type with one of the error messages.